app/assets/src/components/ui/controls/HostOrganismSearchBox.jsx

```diff
@@ -0,0 +1,83 @@
+import React from "react";
+import { get } from "lodash/fp";
+
+import PropTypes from "~/components/utils/propTypes";
+import {
+  matchType,
+  sortTypes,
+} from "~/components/views/SampleUploadFlow/utils";
+
+import LiveSearchPopBox from "./LiveSearchPopBox";
+
+// Adapted from SampleTypeSearchBox
+class HostOrganismSearchBox extends React.Component {
+  handleSearchTriggered = query => {
+    return this.buildResults(this.getMatchesByCategory(query), query);
+  };
+
+  // TODO (gdingle): extract all these to utils if re-used
+  getMatchesByCategory(query) {
+    const matchedHostGenomes = this.props.hostGenomes.filter(
+      hostGenome =>
+        // IMPORTANT NOTE: Only existing, null-user host genomes will be shown as
+        // options for new samples until the team gets a chance to review this
+        // policy in light of the data. See HostGenome.rb.
```

Comment:
should we have a JIRA and date for this note?

Reply:
Good idea. @oliviabholmes do you want to create a JIRA or should I?